### ISSUE TITLE
Add "Reason Transferred" to output - Enhancement to keep the best mon's

### ISSUE THREAD

**Issue by xafelld** [NECROBOTIO/NecroBot#3379]
A lot of folks are starting to tweak the transfer settings. Would it be possible to incluse the reason a mon was transferred in the text output?
Example:  (TRANSFERED) Pidgey - CP: 123  IV: 57.61%   [Global - CP too low] [Best CP: 234  IV: 79.52%] (Candies: 57)
Example:  (TRANSFERED) Pidgey - CP: 123  IV: 57.61%   [Global - IV too low] [Best CP: 234  IV: 79.52%] (Candies: 57)
Example:  (TRANSFERED) Pidgey - CP: 123  IV: 57.61%   [Xfer Filter - Level too low] [Best CP: 234  IV: 79.52%] (Candies: 57)

Basically some what to know if the mon was transfers due to a global setting or a transfer filter setting and then what specific stat triggered the transfer.
This would let people tweak and test and be able to fine what they keep. Once accounts are level ~28+ is is time to start keeping the best mons!

**Comment by Menton** [comment]
I like this idea!
